sql_generators/active_users_aggregates_v3/__init__.py

```diff
@@ -11,10 +11,9 @@
 from bigquery_etl.util.common import render, write_sql
 
 THIS_PATH = Path(os.path.dirname(__file__))
-TABLE_NAME = "active_users_aggregates"
+TABLE_NAME = os.path.basename(os.path.normpath(THIS_PATH))
+BASE_NAME = "_".join(TABLE_NAME.split("_")[:-1])
 DATASET_FOR_UNIONED_VIEWS = "telemetry"
-DESKTOP_TABLE_VERSION = "v1"
-MOBILE_TABLE_VERSION = "v3"
```

Comment:
Will this break everything or mess things up for Desktop AUA? If I'm reading the updated code in this PR correctly it looks like both Desktop and Mobile AUA will get 'v3', even though it seems like currently Desktop is only on v1.

Maybe this isn't an issue because anything that uses these "versioned" AUA tables always grabs the latest version? So 'v1' going straight to 'v3' with no actual changes to the table or schema is just an under-the-hood accounting thing? And it looks like the upsides of this change are that things are more standardized and it'll be easier to track meaningful version changes going forward?

Reply:
I think it's okay if the tables are not sequential, since this change makes it clear which `sql_generator` generated the table.

Reply:
It wouldn't break as the view pick the latest version. This standardizes and should facilitate implementing shredder impact mitigation.